docs/source/data/resolvers.mdx

```diff
@@ -151,17 +153,12 @@ const resolvers = {
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  csrfPrevention: true,
-  cache: 'bounded',
-  plugins: [
-    ApolloServerPluginLandingPageLocalDefault({ embed: true }),
-  ],
 });
 
 // Launch the server
-server.listen().then(({ url }) => {
-  console.log(`🚀  Server ready at ${url}`);
-});
+const { url } = await startStandaloneServer(server, { listen: { port: 4000 } });
```

Comment:
that's the default listen argument, so you can just do `await startStandaloneServer(server)` (this applies in many places).

Reply:
Great point, I'll go through and do that across the board for consistency